ui/pages/confirmations/components/confirm/header/header-info.test.tsx

```diff
@@ -75,5 +76,37 @@ describe('Header', () => {
         },
       });
     });
+
+    it(`sends "${MetaMetricsEventName.AccountDetailsOpened}" metametric for a transaction`, () => {
+      const mockTransactionStore = {
+        metamask: {
+          ...mockState.metamask,
+        },
+        confirm: {
+          currentConfirmation: {
+            type: 'a_transaction_type',
+          },
+        },
+      };
+      const mockTrackEvent = jest.fn();
+      const { getByLabelText } = renderWithProvider(
+        <MetaMetricsContext.Provider value={mockTrackEvent}>
+          <HeaderInfo />
+        </MetaMetricsContext.Provider>,
+        configureStore(mockTransactionStore),
+      );
+      const accountInfoIcon = getByLabelText('Account details');
+      fireEvent.click(accountInfoIcon);
+
+      expect(mockTrackEvent).toHaveBeenNthCalledWith(1, {
+        category: MetaMetricsEventCategory.Transactions,
+        event: MetaMetricsEventName.AccountDetailsOpened,
+        properties: {
+          action: 'Confirm Screen',
+          location: MetaMetricsEventLocation.Transaction,
+          transaction_type: 'a_transaction_type',
+        },
+      });
+    });
```

Comment:
both above tests look largely same, is it possible to loop over confirmation objects ?

Reply:
done